docs/en/your-profile-and-preferences/reset-your-notification-settings.md

```diff
@@ -0,0 +1,28 @@
+---
+id: 640
+revision: 0
+language: en
+hide:
+  - navigation
+---
+
+# Reset your notification settings
+
+!!! note ""
+    Currently, you can only reset your notifications settings using Status mobile.
+
+You can reset your notifications settings if you have customized your notifications, but you have lost track (your notifications don't work the way you want). When you reset your notifications, your previous configuration is discarded and your notifications settings return to the default configuration.
+
+## Reset your notification settings
+
+Resetting notifications settings for communities, channels, group chats, and direct messages (DM) is similar. Follow the steps below for each one of them.
```

Comment:
You can reset your notification settings for communities, channels, group chats and direct messages (DM). Here's how: